subsys/net/lib/nrf_cloud/CMakeLists.txt

```diff
@@ -10,11 +10,9 @@ zephyr_library_sources(
 	src/nrf_cloud_fsm.c
 	src/nrf_cloud_transport.c
 	src/nrf_cloud_sanity.c
-)
-zephyr_library_sources_ifdef(
-	CONFIG_NRF_CLOUD_AGPS
```

Comment:
Why is this included unconditionally now? Aren't these files still depending on `CONFIG_NRF_CLOUD_AGPS`?

Reply:
The same source file nrf_cloud_agps.c is required for both CONFIG_NRF_CLOUD_CELL_POS and CONFIG_NRF_CLOUD_AGPS now, while in real case only one of the flags would be enabled. It's too much to do in order to separate the cell positioning from A-GPS in two source files.

Reply:
Yes but you could still include the file conditionally:
```
if(CONFIG_NRF_CLOUD_AGPS OR CONFIG_NRF_CLOUD_CELL_POS)
  zephyr_library_sources(
    src/nrf_cloud_agps.c
    src/nrf_cloud_agps_utils.c
  )
endif()
```

Reply:
Did not know this way of OR and had tried to find out how.
Taken with gratitude.